.env.example

```diff
@@ -42,4 +42,5 @@ DISABLE_DEV_REDUX_LOGGER=
 # [Optional] disable device logger during development. Enabled by default, set "1" to disable
 DISABLE_DEV_DEVICE_LOGGER=
 
-
+# [Optional] set desired latest release. Is based on enum OsEnvironment, so can be set to 'production', 'test-production' or 'daily'.
+FEATURE_TOGGLE_LATEST_RELEASE=
```

Comment:
I think we should call it similar to FEATURE_TOGGLE_ENVIROMENT so something like FEATURE_TOGGLE_RELEASE_ENVIROMENT

Reply:
sure :D I didn't have good idea for it :D